src/docs/product/performance/transaction-summary.mdx

```diff
@@ -94,6 +105,42 @@ The **Tags** tab displays a list of suspect tag keys that often correspond to sl
 
 The table in **All Events** tab offers a full list of transactions broken down similar to the abbreviated table in the overview. Filters are carried across from other views. You can also narrow the events down by percentile.
 
+## Spans
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The "Spans" tab displays a list of suspect spans that correspond to where most of the time in a transaction is being spent and can help you find spans where performance problems lie.
```

Comment:
```suggestion
The "Spans" tab displays a list of suspect spans that correspond to where most of the time in a transaction is being spent, and it can help you find spans where performance problems lie.
```